packages/eslint-config-eslint/package.json

```diff
@@ -21,7 +50,8 @@
     "cjs.js",
     "formatting.js",
     "index.js",
-    "nodejs.js"
+    "nodejs.js",
+    "types"
```

Comment:
We don't need to include `ambient-module-declarations`?

Reply:
No. The `ambient-module-declarations` is just for us.